src/index.js

```diff
@@ -517,6 +656,20 @@ function highlightMove (id) {
         }
       } catch (e) {}
       // En Pasant
+      try {
+        if (boardSquare[rightId].isEmpty === false && getFigureById(rightId).enPassant === true) {
+          if (element.color === 'white') {
+            movesDraw(topRightId, 'top-right', element, 1);
+          } else movesDraw(bottomRightId, 'bottom-right', element, 1);
+        }
+      } catch (e) {}
```

Comment:
but why???

Reply:
Sometimes 'rightId' may not exist and boardSquare[rightId].isEmpty will lead to an error: Trying to find ''isEmpty' in undefined

Reply:
so maybe
`if (rightId && boardSquare[rightId].isEmpty === false && getFigureById(rightId).enPassant === true) {`